### PR TITLE
Various cleanups

### PR DESCRIPTION
**Pull request by michaelforney**

### CODE REVIEW

File: FM-Net/fm-net.c [hunk 1]
```diff
@@ -169,48 +217,63 @@ void rewrite(Net* net, u32 a_addr) {
...
-        case 14: res = Numeric(fst < snd ? 1 : 0); break;
-        case 15: res = Numeric(fst == snd ? 1 : 0); break;
-        default: res = 0; printf("[ERROR]\nInvalid interaction."); break;
+      case ADD: res.i = fst.i + snd.i; break;
+      case SUB: res.i = fst.i - snd.i; break;
+      case MUL: res.i = fst.i * snd.i; break;
+      case DIV: res.i = fst.i / snd.i; break;
+      case MOD: res.i = fst.i % snd.i; break;
+      case POW: res.i = powi(fst.i, snd.i); break;
+      case AND: res.i = fst.i & snd.i; break;
```
Comment (bamorim) [inline FM-Net/fm-net.c:236]: I guess the kind 6 wasn't supposed to be the AND, but rather fst^(snd/2^32), whatever that was supposed to be, right? Don't this break anything? Would be nice to add some regression testing.

**Review by bamorim — COMMENTED** [review]
For now, I liked that you added clang-format to this. But I'm just skeptical about if the changes break anything.
IMHO, maybe before we change anything we should add some kind of testing. Maybe we could run the net in the JS compiler or something like that, so we ensure the C runtime has the same behaviour as the JS one. Don't know.

@MaiaVictor has any thoughts on that?

File: FM-Net/fm-net.c (hunk 1 above)
Comment (michaelforney) [reply FM-Net/fm-net.c:236]: If you look at the individual commits, they first switch to using the enum (https://github.com/moonad/Formality-C/pull/2/commits/a6ae49f0376248a8f0e4c281579032c4f42efc9c), and then switch to the op codes used in the latest Formality (https://github.com/moonad/Formality-C/pull/2/commits/05fb45454a17bfda84d09d76059ed7ad34d2f87c), which does *not* have that operator.

In the latter commit description, I referred to the corresponding Formality commit, which you can see also removes that operator:
https://github.com/moonad/Formality/commit/94fb510db029ec86ac815b1c57675763ab520f29#diff-5eb486a0616553404e3b1f2431a54e23

**Comment by michaelforney** [comment]
Are there any changes besides the op codes that you are concerned about? The others are just stylistic changes. As far as testing, before we can compare the results of the C runtime to the JS runtime, they need to use the same op codes so that they evaluate the net in the same way.

**Comment by VictorTaelin** [comment]
@bamorim those changes look fine to me. I'd like some tests here, too. But nothing depends on Formality-C so it is fine to experiment.

**Pull request merged**